### ISSUE TITLE
[FEATURE] Automatik für Text-to-PDF

### ISSUE THREAD

**Issue by Dickmaulruessler** [jlawyerorg/j-lawyer-org#1933]
Ich wandle - ausnahmslos jedes - meiner Textdokument manuell in PDF um und schiebe es danach in einen anderen Ordner. 
Ich finde es ist dann später einfacher und praktischer schnell durch die Akte zu wandern und Dokumente zu lesen.

Machen das viele so? 

Falls ja und wenn machbar: Wäre das perspektivisch irgendeine Automation möglich? Also zum Beispiel dass nach dem beA-Versand irgendeine Automatik läuft inkl. Verschieben des Textdokuments in einen Unterordner?

Ja, ich weiß, das wäre ganz schön viel Luxus und ich glaube es wäre auch viel Arbeit ..... aber ich frage mal....

**Comment by Thorsten-Nutzer** [comment]
Ja, ich versuche auch, alle Dokumente in eine archivierbare Form zu bringen, die man später ruck-zuck noch öffnen kann. Das gilt auch und gerade für E-Mails und beAs.
Mit Thunderbird kann ich E-Mails als PDFs ausdrucken. Alternativ gibt es ein Add-On für Thunderbird "ImportExportTools NG" mit denen man E-Mails noch komfortabler als PDFs exportieren kann. Beides ganz gut - aber eben eine externe Lösung.

**Comment by Mucki526** [comment]
Ich lege jedes Dokument als PDF ab und blende das Libreoffice-Schreiben per Ordner aus. Wenn ich mich in der Akte orientiere, dann an den PDF. Dito mit den BEA. Die gehen in den Ordner, was ich brauche (PDF) wird ausgepackt), der Rest bei Bedaruf ausgeblendet. Das gleiche mit e-mails. Am Ende des Tages ist die Idee, alle relevanten Dokumente als PDF zu sehen, sonst nichts.  So mache ichs... die Word-Dokumente können zu leicht verfälscht werden. Die PDF sind da etwas "schreibgeschützter". Das importExporttools wünsche ich mir auch als Brücke von Thunderbird ins j-lawyer.

**Comment by stm9x9** [comment]
> Also zum Beispiel dass nach dem beA-Versand irgendeine Automatik läuft inkl. Verschieben des Textdokuments in einen Unterordner?

Wieso nach dem beA-Versand? Ich wandle meine Texte erst in pdf um (signiere sie dann) und versende dann das pdf!?

**Comment by Dickmaulruessler** [comment]
> 
> Wieso nach dem beA-Versand? Ich wandle meine Texte erst in pdf um (signiere sie dann) und versende dann das pdf!?

Dadurch sind es insgesamt doch aber nicht weniger Schritte...
